beesdoo_pos/static/src/js/beesdoo.js

```diff
@@ -1,7 +1,7 @@
 odoo.define('beescoop.pos', function (require) {
     "use strict";
     var module   = require("point_of_sale.screens");
```

Comment:
rename `module` occurrences in `screens`
`var screens = require("point_of_sale.screens");`